src/Common/src/Interop/Windows/NtDll/Interop.RtlGetVersion.cs

```diff
@@ -11,12 +11,17 @@ internal partial class NtDll
         [DllImport(Libraries.NtDll, ExactSpelling=true)]
         private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOEX lpVersionInformation);
 
-        internal static unsafe string RtlGetVersion()
+        internal static unsafe int RtlGetVersionEx(out RTL_OSVERSIONINFOEX osvi)
         {
-            var osvi = new RTL_OSVERSIONINFOEX();
+            osvi = new RTL_OSVERSIONINFOEX();
             osvi.dwOSVersionInfoSize = (uint)sizeof(RTL_OSVERSIONINFOEX);
+            return RtlGetVersion(ref osvi);
+        }
+
+        internal static unsafe string RtlGetVersion()
```

Comment:
Ditto

Reply:
Again, it's the fixed size buffer. Unsure if to use it without pointer.

Reply:
Yup, I missed that sneaky `&`. Thanks.